.rhcicd/clowdapp-backend.yaml

```
@@ -320,6 +324,10 @@ parameters:
 - name: SENTRY_ENABLED
   description: Enable Sentry (or not)
   value: "false"
+- name: QUARKUS_OPENTELEMETRY_ENABLED
+  value: "false"
+- name: QUARKUS_OPENTELEMETRY_ENDPOINT
+  value: "localhost:4317"
```

I'm not sure if that's mandatory:
```suggestion
  value: "http://localhost:4317"
```